src/sidebar/components/SidebarTabbedPanel.tsx

```diff
@@ -82,17 +91,10 @@ function Tab({
   );
 }
 
-export type SelectionTabProps = {
+export type SidebarTabbedPanelProps = {
   /** Are we waiting on any annotations from the server? */
   isLoading: boolean;
 
-  /** Counts of threads in each tab, to be displayed next to the tab title. */
-  tabCounts: {
-    annotation: number;
-    note: number;
-    orphan: number;
-  };
```

Comment:
A benefit of calling `useRootThread();` here, is that passing `tabCounts` is no longer needed.